src/webapp/hooks/useImportExportTranslation.ts

```diff
@@ -0,0 +1,43 @@
+import { useCallback } from "react";
+import _ from "lodash";
+import JSZip from "jszip";
+import FileSaver from "file-saver";
+import i18n from "../../locales";
+import { useSnackbar } from "@eyeseetea/d2-ui-components";
+
+export function useImportExportTranslation() {
+    const snackbar = useSnackbar();
+
+    const exportTranslation = useCallback(
+        async (exportFn: () => Promise<Record<string, Record<string, string>>>, name: string) => {
```

Comment:
A more declarative type as mentioned above for the translations could increase readability.

Reply:
Yes, "string" is too generic when used with untagged containers (Record, Map), we can create type aliases to give names to types.